synapse/config/experimental.py

```diff
@@ -71,6 +71,9 @@ def read_config(self, config: JsonDict, **kwargs: Any) -> None:
         self.msc3720_enabled: bool = experimental.get("msc3720_enabled", False)
 
         # MSC2654: Unread counts
+        #
+        # Note that when this is enabled previously calculated push rules might
```

Comment:
s/push rules/push actions/?

Also I feel like this sentence could use a bit of punctuation to make it easier to read (it took me a few reads to realise that "previously" relates to "calculated" not "enabled", if that makes sense).

Reply:
I reworded the sentence to clarify this.